modules/pruning-images-manual.adoc

```diff
@@ -5,20 +5,91 @@
 [id="pruning-images-manual_{context}"]
 = Manually pruning images
 
-The Pruning Custom Resource enables automatic image pruning. However, administrators can manually prune images that are no longer required by the system due to age, status, or exceed limits by running the following command:
+The Pruning Custom Resource enables automatic image pruning. However, administrators can manually prune images that are no longer required by the system due to age, status, or exceed limits. There are two methods to manually prune images:
 
-[source,terminal]
+* Running image pruning as a `Job` or `CronJob` on the cluster.
+* Running the `oc adm prune images [<options>]` command.
+
+.Prerequisites
+
+* To manually prune images, you must first log in to the CLI as a user with an access token. The user must also have the cluster role of `system:image-pruner` or greater (for example, `cluster-admin`).
```

Comment:
Do we need To manually prune images, here?
Seems a bit confusing to me. Sounds like you need this role for one of the two methods, which isn't true(?).